strawberry/printer/printer.py

```diff
@@ -161,6 +162,30 @@ def print_field_directives(
     )
 
 
+def print_args(args: Dict[str, GraphQLArgument], indentation: str = "") -> str:
+    if not args:
+        return ""
+
+    # If every arg does not have a description, print them on one line.
+    if not any(arg.description for arg in args.values()):
+        return (
+            "("
+            + ", ".join(print_input_value(name, arg) for name, arg in args.items())
+            + ")"
+        )
+
+    return (
+        "(\n"
+        + "\n".join(
+            print_description(arg, f"  {indentation}", not i)
+            + f"  {indentation}"
+            + print_input_value(name, arg)
+            for i, (name, arg) in enumerate(args.items())
+        )
+        + f"\n{indentation})"
+    )
```

Comment:
Do you think it would be better to split this into normal `for` loops instead of a really dense one-liner?

Reply:
yes, but I also want to rewrite the whole printer to not use any GraphQL core code